Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/french/parsers/FrenchTimeParserConfiguration.java

```diff
@@ -101,7 +101,7 @@ public PrefixAdjustResult adjustByPrefix(final String prefix, int hour, int min,
         }
 
         // 'to' i.e 'one to five' = 'un à cinq'
-        if (trimmedPrefix.endsWith("à")) {
+        if (trimmedPrefix.endsWith("à") || trimmedPrefix.contains("moins")) {
```

Comment:
I know refactoring the Java code is not currently high priority, but could you add a '// @TODO Mode hardcoded strings to resource YAML file.' comment at the modified points?

Reply:
ok, added